### PR TITLE
Add password reset link to login page

### PR DESCRIPTION
**Pull request by ethanhs**
Fixes rt#6836

### CODE REVIEW

File: ocfweb/login/urls.py [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
+from ocfweb.account.urls import urlpatterns as account
 from ocfweb.login.calnet import login as calnet_login
 from ocfweb.login.calnet import logout as calnet_logout
 from ocfweb.login.ocf import login
 from ocfweb.login.ocf import logout
 
 
 urlpatterns = [
+    url(r'^account/', include(account)),
+
```
Comment (jvperrin) [inline ocfweb/login/urls.py]: Why do all these changes need to be made? I believe the link will just work without all of this.

**Comment by jvperrin** [comment]
I was messing with this a bit before, and I thought putting the link in a position like this would work pretty well.  I thought it looked nice having the link in the bottom right since it worked well with the login button, but it could probably be shifted down a bit so that it aligns with the login button vertically instead of floating a bit above it:

reset-link-position

I did that with the following (which I think should be cleaned up a bit to not use as much inline CSS):

    <div style="max-width: 300px;">
        {{ form|bootstrap }}
        <small style="float: right;">
            <a href="{% url 'change_password' %}">Forgot your password?</a>
        </small>
    </div>

File: ocfweb/login/urls.py (hunk 1 above)
Comment (ethanhs) [reply ocfweb/login/urls.py]: Oh, you are right, it is already included in the main urlpatterns.

**Comment by ethanhs** [comment]
@jvperrin I got it to shift down nicely if I add `margin-top: 6pt;`.
image

However, is it worth it to add the complexity of a new scss for login for just one dom element?

File: ocfweb/login/templates/login/ocf/login.html [hunk 2]
```diff
@@ -19,9 +19,12 @@
                     {% csrf_token %}
                     <div style="max-width: 300px;">
                         {{ form|bootstrap }}
+            <small class="reset-link">
+                <a href="{% url 'change_password' %}">Forgot your password?</a>
+            </small>
```
Comment (chriskuehl) [inline ocfweb/login/templates/login/ocf/login.html]: indented weird (plus, hard tabs?)

Comment (ethanhs) [reply ocfweb/login/templates/login/ocf/login.html]: Hm, that is strange I'll fix that. When you say hard tabs, are you suggesting I use them in future or something else?

Comment (chriskuehl) [reply ocfweb/login/templates/login/ocf/login.html]: yeah, we generally use 4 space indents at the ocf (unless the language has a well-known convention, e.g. overwhelming puppet convention is 2 space indents, so we use that)

**Comment by ethanhs** [comment]
Any more changes needed for this?

**Comment by jvperrin** [comment]
Nope, I think that's it! I've sent you an invitation to join to OCF organization, then you should be able to merge your own PR. Thanks for the change!

**Comment by ethanhs** [comment]
Thank you @jvperrin!

**Pull request merged**